src/github/workflows/deploy.yml

```diff
@@ -294,4 +294,4 @@ jobs:
       run: |
         gh release delete 'nightly' --repo agda/agda --cleanup-tag --yes || true
         ls -R artifacts
-        gh release create 'nightly' artifacts/**/* --repo agda/agda --generate-notes --title "Nightly Build (${{ needs.build.outputs.sha }}@master)"
+        gh release create 'nightly' artifacts/**/* --repo agda/agda --generate-notes --notes-start-tag v2.6.3 --title "Nightly Build (${{ needs.build.outputs.sha }}@master)"
```

Comment:
I added an item to the release procedure in commit 49d93817f30032eab1d733b2dc367b19831f6092 because the tag version is hard-coded.

Reply:
Ah, thanks.